gitlab-ci: fixup evaluator results file with advisor results

unclejay80 opened the pull request:
Evaluator should use the advisor results although the scanner is deactivated

Signed-off-by: Jürgen Wischer <33184634+unclejay80@users.noreply.github.com>

Please ensure that your pull request adheres to our [contribution guidelines](https://github.com/oss-review-toolkit/.github/blob/main/CONTRIBUTING.md). Thank you!

sschuberth commented on scripts/ort-evaluator.sh:
```diff
@@ -4,13 +4,16 @@
     ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_SCANNER_FILE
 fi
 
+
```
Nit: Unrelated change.

sschuberth replied:
This unrelated line change is still present in the diff.

unclejay80 replied:
should be fine now

sschuberth commented on scripts/ort-evaluator.sh:
```diff
@@ -4,13 +4,16 @@
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ANALYZER_FILE
-elif [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
-    ORT_RESULTS_INPUT_FILE=$ORT_RESULTS_ADVISOR_FILE
+    if [[ "$ORT_DISABLE_ADVISOR" = "false" ]]; then
```
Nit: Let's stick to the same logic as one line above and compare with `= "true"`, swapping the code branches below.

sschuberth replied:
I guess something went wrong while addressing this: The change per se is fine with me and should stay. Just the order of the introduced branches in the nested condition should be swapped, to match the logic of the surrounding condition.

unclejay80 replied:
next try :) I hope this is fine now

sschuberth replied:
Just some commit message tweaking 😉 Please use:

```
ort-evaluator: Use the correct results file

Use the advisor results file if the scanner is disabled but the advisor is
enabled.
```

(Plus your `Signed-of-by`, of course.)

unclejay80 replied:
done

unclejay80 committed:
ort-evaluator: Use the correct results file

Use the advisor results file if the scanner is disabled but the advisor is
enabled.

Signed-off-by: Jürgen Wischer <juergen.wischer@bwi.de>
